crates/sc-consensus-subspace-rpc/src/lib.rs

```diff
@@ -154,6 +155,16 @@ pub trait SubspaceRpcApi {
 
     #[method(name = "subspace_lastSegmentHeaders")]
     async fn last_segment_headers(&self, limit: u64) -> Result<Vec<Option<SegmentHeader>>, Error>;
+
+    /// Block/transaction archived object mappings subscription
+    #[subscription(
+        name = "subspace_subscribeArchivedObjectMappings" => "subspace_archived_object_mappings",
+        unsubscribe = "subspace_unsubscribeArchivedObjectMappings",
+        item = GlobalObjectMapping,
```

Comment:
I just now noticed this. You seem to assume that all object mappings will fit into a single RPC message, which is very likely to not be the case. I was expecting it to stream individual mappings as separate messages or maybe in batches, but definitely not all at once from the whole segment, that is not going to work.

Reply:
This was my quick fix to having large numbers of messages, but I can flip it back to the way it was. See commit 7363a783a1.

I'll work out how to do batches or feedback once I've done some manual testing. The default RPC limits are 10MB responses and 1024 items, does subspace change that?

Reply:
We use Substrate's defaults right now: https://github.com/autonomys/subspace/blob/aa2a50b5106ff6ab6ee356641ea8b007c7e787a9/crates/subspace-node/src/commands/run/shared.rs#L10